### ISSUE TITLE
`.git` directory inconsistencies on initial clone

### ISSUE THREAD

**Issue by willrowe** [sublimehq/sublime_merge#882]
**Version info**

 - OS: macOS 10.15.6
 - Build: 2031
 - Git Binary: bundled
 - System Git: 2.24.3

**Description**

When I clone a repository using SM and then look at the `.git` directory that is generated I see this:
```
HEAD
config
index
logs/
objects/
packed-refs
refs/
```

However, if I clone the same repository from the command line using the system git I see this:
```
HEAD
config
description
hooks/
index
info/
logs/
objects/
packed-refs
refs/
```

Most noticeably the `hooks` and `info` directories are missing when SM generates the directory.

**Steps to reproduce**

1. Clone a repository.
2. Look at the files in the generated `.git` directory.

**Expected behavior**

I would expect the `.git` directory to be the same in both cases using the configured git template directory.

**Comment by dpjohnst** [comment]
Hi @willrowe,

Thanks for reporting this issue - it's likely that the default template directory that comes with our bundled Git does not match the system Git you have installed. I'll be investigating this further.

Just to confirm, do you have any `init.templateDir` git configuration variable set?

Thanks,
\- Dylan

**Comment by willrowe** [comment]
I just double-checked and that configuration variable is not set in any of the configuration files.

**Comment by willrowe** [comment]
I just tested this using build 2043 and it appears to be working as expected now. I'm not sure if this was fixed in SM or if something else changed, but I will close this  for now since it appears to have been resolved.

**Comment by dpjohnst** [comment]
Hi @willrowe,

I believe this was fixed with the upgrade of the bundled Git to 2.29.2 in build 2041.

Cheers,
\- Dylan
